### PR TITLE
RAC-1103: Fix glitches + add comments

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description (for Contributor and Core Developer)**

Fix small glitches after review in https://github.com/akeneo/pim-community-dev/pull/15818/files#

**Definition Of Done (for Core Developer only)**

- [x] Tests

### CODE REVIEW

**Review by ghoussard — APPROVED** [review]

File: src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php [hunk 1]
```diff
@@ -23,14 +23,21 @@ public function __construct(Connection $connection)
...
+    private function createSqlQuery(FindJobUsersQuery $query): string
     {
         $username = $query->search;
 
         $sql = <<<SQL
             SELECT DISTINCT job_execution.user
             FROM akeneo_batch_job_execution job_execution
             WHERE job_execution.is_visible = 1
-            %s
+            %%%s%%
```
Comment (ghoussard) [inline src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php:40]:
```suggestion
            %s
```

Comment (ghoussard) [reply src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php:40]: You should wrap ':username' with % on line 47 only

**Review by ghoussard — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx [hunk 2]
```diff
@@ -187,3 +187,8 @@ test('it can change page', () => {
   expect(screen.queryByText('Export job')).not.toBeInTheDocument();
   expect(screen.getByText('Mass edit')).toBeInTheDocument();
 });
+
+// TODO: should we test that when changing a filter:
+//  - the grid updates
+//  - the count updates
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx:193]: Our test are stateless for me we didn't need to test if we test that the hook is called with the right information and we test that the right information is displayed given a scenario

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx [hunk 3]
```diff
@@ -187,3 +187,8 @@ test('it can change page', () => {
...
+//  - the pagination does not change (?)
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx:194]: The pagination is reset to 1 when changing a filter it's something we tested

**Review by StevenVAIDIE — APPROVED** [review]

File: src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx (hunk 3 above)
Comment (SamirBoulil) [reply src/Akeneo/Platform/Job/front/process-tracker/src/feature/pages/JobExecutionList.test.tsx]: removed

File: src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php (hunk 1 above)
Comment (SamirBoulil) [reply src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php]: fixed

Comment (SamirBoulil) [reply src/Akeneo/Platform/Job/back/Infrastructure/Query/FindJobUsers.php]: Not fixed, this way breaks the tests (the query is not interpreted the same by the DBAL) :'(

**Pull request merged**